web-client/public/js/grnstate.js

```diff
@@ -31,16 +32,21 @@ export const grnState = {
 
     newNetwork: false,
 
+    normalizationMax: null,
```

Comment:
I'm observing that `normalizationMax` does not appear to be touched in this round, but we can wait-and-see for this one.